src/windowsservercore/ltsc2022/helix/webassembly-net8/amd64/Dockerfile

```diff
@@ -41,12 +41,14 @@ RUN mkdir %EMSCRIPTEN_PATH% `
     && .\upstream\emscripten\embuilder.bat build MINIMAL
 
 # Install latest 18.x Node JS
+ENV NODE_RELEASE 18
```

Comment:
Docker recommends using `ENV NODE_RELEASE=18` syntax instead. The space syntax may be deprecated at some point.

Reply:
fixed in https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/1257/commits/260d729f91984aed7997917f54f3161892ee639b